ui/src/reusable_ui/components/wizard/WizardButtonBar.scss

```diff
@@ -0,0 +1,16 @@
+@import 'src/style/modules/variables';
+
+.button-bar {
```

Comment:
I suggest namespacing this to something like `.wizard-button-bar` to avoid potential collisions since `button` and `bar` are fairly generic names